server/sonar-web/src/main/js/components/issue/components/IssueActionsBar.js

```diff
@@ -84,6 +77,13 @@ export default class IssueActionsBar extends React.PureComponent {
     this.props.togglePopup('comment', open);
   };
 
+  handleTransition = (issue /*: Issue */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)